pkg/opensearch/response_parser_test.go

```diff
@@ -1988,10 +2344,46 @@ func TestProcessRawDocumentResponse(t *testing.T) {
 	})
 }
 
-func Test_sortPropNames_puts_timeField_at_the_beginning(t *testing.T) {
-	actual := sortPropNames(
-		map[string]bool{"_id": true, "Average": true, "timestamp": true},
-		client.ConfiguredFields{TimeField: "timestamp"},
-	)
-	assert.Equal(t, []string{"timestamp", "Average", "_id"}, actual)
+func Test_sortPropNames(t *testing.T) {
+	t.Run("returns slice after finding fields in prop names and placing them in front", func(t *testing.T) {
+		actual := sortPropNames(
+			map[string]bool{"lookForThisField": true, "_another_field": true},
```

Comment:
maybe slightly more obvious if we switch the order? `map[string]bool{"_another_field": true, "lookForThisField": true },`